suite-native/navigation/src/components/TabBarItem.tsx

```diff
@@ -32,14 +44,17 @@ export const TabBarItem = ({ isFocused, onPress, iconName, title }: TabBarItemPr
              */
                 onPress();
             }}
+            hitSlop={{ top: hitSlop, right: hitSlop, bottom: hitSlop, left: hitSlop }}
```

Comment:
Could be just number if all sides are same size, but I think we should not need right and left slop, because you should be able to style it using flex that touchable areas will be that big that their are will be next to each other

Reply:
Agree with right and left slop. I have also removed it from props and moved it just to the constant for horizontal slop - 5affd46d13fd9ec17830916b30c01102c0a15b9d